src/statbank/auth.py

```diff
@@ -54,12 +40,13 @@ def check_env() -> str:
         """
         return os.environ.get("DAPLA_ENVIRONMENT", "TEST")
 
-    @staticmethod
-    def check_database() -> str:
+    def check_database(self) -> str:
         """Checks if we are in prod environment. And which statbank-database we are sending to."""
         db = "TEST"
         if os.environ.get("DAPLA_ENVIRONMENT", "TEST") == "PROD":
             db = "PROD"
+        if self.use_test_db:  # type: ignore[attr-defined]
```

Comment:
Variablene som brukes i StatbankAuth bør settes i `StatbankAuth.__init__()` og kalles opp med `super().__init__()` fra `StatbankClient.__init__()`

Reply:
Since Uttrekk has a double inheritance, I will call with StatbankAuth.__init__ in the code to stay consistent.